src/main.cpp

```diff
@@ -3988,8 +3988,17 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
         return true;
     }
 
-
-
+    if ((strCommand == "filterload" ||
```

Comment:
How?

Please explain things like this in your commit messages. Why you did it, why it is correct, and so on.
Say - this causes a bug at some point in the future, developers need to be able to know why you did it. With an empty commit message (apart from the title) that's hard to figure out.